tests/e2e/helpers/searchTestParams.js

```diff
@@ -0,0 +1,4 @@
+export const TEST_TEXT_FOUND = 'theory';
```

Comment:
These might better live inside mockSearch.js; not important enough to need their own file :)